rest/taskana-rest-spring/src/test/java/pro/taskana/ldap/LdapClientTest.java

```diff
@@ -0,0 +1,60 @@
+package pro.taskana.ldap;
+
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.core.env.Environment;
+import org.springframework.ldap.core.LdapTemplate;
+
+class LdapClientTest {
+
+  @Mock Environment environment;
+
+  @Mock LdapTemplate ldapTemplate;
+
+  @InjectMocks LdapClient cut;
+
+  @Test
+  void testLdap() {
+
+    MockitoAnnotations.initMocks(this);
```

Comment:
can we please use the @ExtendWith(MockitoExtension.class) annotation?

Reply:
like we always do :)

Reply:
@benjamineckstein this is still not done :)

Reply:
done